index.d.ts

```diff
@@ -1,18 +1,25 @@
 declare namespace publicIp {
 	interface Options {
 		/**
-		Use a HTTPS check using the [icanhazip.com](https://github.com/major/icanhaz) service instead of the DNS query. This check is much more secure and tamper-proof, but also a lot slower. **This option is only available in the Node.js version**.
+		Use a HTTPS check using the [icanhazip.com](https://github.com/major/icanhaz) service instead of the DNS query. [ipify.org](https://www.ipify.org) is used as a fallback if `icanhazip.com` fails. This check is much more secure and tamper-proof, but also a lot slower. **This option is only available in the Node.js version**. Default behaviour is to check aginst DNS before using HTTPS fallback, if set as `true` it will *only* check against HTTPS.
 
-		@default false
+		@default null
 		*/
-		readonly https?: boolean;
+		readonly onlyHttps?: boolean;
 
 		/**
 		The time in milliseconds until a request is considered timed out.
 
 		@default 5000
 		*/
 		readonly timeout?: number;
+
+		/**
+		Use a HTTPS check using the [icanhazip.com](https://github.com/major/icanhaz) service instead of the DNS query. [ipify.org](https://www.ipify.org) is used as a fallback if `icanhazip.com` fails. This check is much more secure and tamper-proof, but also a lot slower. **This option is only available in the Node.js version**. Default behaviour is to check aginst DNS before using HTTPS fallback, if set as `true` it will *only* check against HTTPS.
```

Comment:
This description is just copy-pasted from `onlyHttps`.

Reply:
I'm sorry I'm not sure what you mean by this. I mean. I know it is, though it should be, and I'm not sure if you would like me to remove it, copy properly or write something dedicated.

Reply:
Why should it be? The text makes no sense in its current form for this option. The text should be about what the `fallbackUrls` option does.

Reply:
This option also needs to be documented in the readme. The readme and index.d.ts should be in sync.

Reply:
my bad ... I see that now :)